vod/manifest_utils.c

```diff
@@ -111,23 +134,23 @@ manifest_utils_build_request_params_string_per_sequence_tracks(
 		result_size += sizeof("-f32") - 1;
 
 		// video tracks
-		if (tracks_mask[MEDIA_TYPE_VIDEO] == 0xffffffff)
+		if (vod_is_all_bits_set(tracks_mask[MEDIA_TYPE_VIDEO], MAX_TRACK_COUNT))
 		{
 			result_size += sizeof("-v0") - 1;
 		}
 		else
 		{
-			result_size += vod_get_number_of_set_bits(tracks_mask[MEDIA_TYPE_VIDEO]) * (sizeof("-v32") - 1);
+			result_size += vod_get_number_of_set_bits_in_mask(tracks_mask[MEDIA_TYPE_VIDEO], MAX_TRACK_COUNT) * (sizeof("-v999") - 1);
```

Comment:
this can be problematic if someone goes crazy and uses MAX_TRACK_COUNT > 1000 + it will allocate unneeded space with the default value of 64. 
I suggest we do something like - 
```
#define MAX_TRACK_INDEX_LEN (sizeof(ngx_value(MAX_TRACK_COUNT)) - 1)
```

Reply:
Nice idea but stringification will include any parentheses and other math expressions https://godbolt.org/z/v5bb9ae1r
`gcc -E` shows whats happening:
```
int sadfasfd() {
    return (sizeof("(12345)") - 1);
}

```

Reply:
Unpopular opinion (not proud): 
```C
#define MAX_TRACK_INDEX_LEN ((int)log10(MAX_TRACK_COUNT) + 1)
```
GCC will optimise this out even with no optimization enabled.
Clang is fine beginning with `-O1`

Reply:
Yeah, I know, I thought about the parenthesis, I'm ok with that, don't like the log10 solution :)

Reply:
Ok, I will add a comment that the value is slightly off